source/basic.tex

```diff
@@ -664,7 +664,8 @@
 D d2;                           // \tcode{X(int, int, int)} called by \tcode{D()};
                                 // \tcode{D()}'s implicit definition violates the ODR
 \end{codeblock}
-\end{example}
+\end{example}%
+; and
```

Comment:
Oh no, not another one of these!

Can we convert all the bullets into separate sentences to avoid this? This is a page-long sentence as-is, and doesn't seem to need to be...

Reply:
Yes, we can.